examples/angular/project.json

```diff
@@ -138,7 +138,7 @@
     "deploy": {
       "executor": "@nrwl/workspace:run-script",
       "options": {
-        "script": "gh-pages -d dist/examples/angular"
+        "script": "gh-pages -d ./dist/examples/angular"
```

Comment:
I don't think there's a need to change the path from an `absolute` to a `relative` here.

Do you have `gh-pages` installed globally? - if so the script will not work for other developers.

We need to install it locally: `yarn add -D gh-pages` and run the commands for building and deploying the angular example which are found in Contributing file.

Reply:
I think we need this, because it is execute from wallet-select/angular so without "./" path would be wallet-select/angular/dist/angular/. 

You can try without it but it will throw "src" error.

Reply:
The command for deploying to gh-pages `yarn nx deploy angular` does not work for me.

![not-found](https://user-images.githubusercontent.com/95851345/191015967-b8ba6725-2d8b-429a-8512-df894f72c298.PNG)

Reply:
Sorry was confused because `yarn.lock` did not change even though you've added `gh-pages` as a devDependency.

Run: `yarn install` this will update yarn.lock and push those changes.

We use `yarn` in this project instead of `npm` so we need to keep that in mind,

Not everyone has those extensions installed the commands in contributing.md should work fine.
I still think the path needs to be an absolute path and you can run these successfully from the root.


```bash
yarn nx build angular # build the angular example
yarn nx deploy angular # deploys angular example to gh-pages
```